src/uu/mv/src/mv.rs

```diff
@@ -399,6 +385,23 @@ fn handle_two_paths(source: &Path, target: &Path, opts: &Options) -> UResult<()>
     }
 }
 
+fn is_subdir(parent: &Path, child: &Path) -> bool {
```

Comment:
Is this different than [`Path::starts_with`](https://doc.rust-lang.org/std/path/struct.Path.html#method.starts_with)? If so, could you add a comment explaining how it's different?

Reply:
Good point, thanks. I fixed it in the latest push.